tests/general/textlint_test_files/bad_alex_no_dead_link.md

```diff
@@ -1,2 +1,2 @@
-We've confirmed his identity.
+We've confirmed their identity.
```

Comment:
Since this is unrelated to the `no_dead_link` test, why not remove it?

Reply:
filling the line allows the line numbers of the other result to stay the same.

It is possible for any content on this line to become an error in the future, as textlint adds more rules, so the text on this line is as good as any other.